ACHNBrowserUI/ACHNBrowserUI/views/settings/AppIconPickerView.swift

```diff
@@ -62,11 +95,13 @@ struct AppIconPickerView: View {
     
     private func setAppIcon(to icon: AppIcon) {
         print("@@ Attempting to set icon: \(icon.name)")
-        if UIApplication.shared.supportsAlternateIcons {
-            UIApplication.shared.setAlternateIconName(icon.name) { (error) in
-                if let error = error {
-                    print("@@ Error setting alternate app icon: \(error)")
-                }
+        
+        self.viewModel.selected = icon.name
+        
+        UIApplication.shared.setAlternateIconName(icon.name) { (error) in
+            if let error = error {
+                print("@@ Error setting alternate app icon: \(error)")
+                self.viewModel.selected = AppIconPickerViewModel.defaultIcon
```

Comment:
I know it makes more sense to just set it *after* it occurs, but for some reason this causes the view to not update properly lol.